onnxruntime/core/framework/bfc_arena.h

```diff
@@ -101,6 +101,11 @@ class BFCArena : public IAllocator {
 
   ArenaType GetArenaType() const { return arena_type_; }
 
+  // Secure the allocated chunk on the target stream
```

Comment:
>// Secure the allocated chunk on the target stream [](http://example.com/codeflow?start=2&length=50)

The comment essentially repeats the function signature, but add no meaning.

Reply:
deleted the comment